doc/nrf/ug_thread_configuring.rst

```diff
@@ -398,9 +374,43 @@ No tick indicates that there is no support for the given feature in the related
       - ✔
       -
       -
+    * - DUA (Thread 1.2)
+      - ✔
+      - ✔
+      - ✔
+      -
+	  * - MLR (Thread 1.2)
+      - ✔
+      - ✔
+      - ✔
+      -
+	  * - BACKBONE_ROUTER (Thread 1.2)
+      - ✔
+      -
+      -
+      -
+	  * - LINK_METRICS_INITIATOR (Thread 1.2)
+      - ✔
+      - ✔
+      - ✔
+      -
+	  * - LINK_METRICS_SUBJECT (Thread 1.2)
+      - ✔
+      - ✔
+      -
+      -
+	  * - CSL_RECEIVER (Thread 1.2)
```

Comment:
The indents on these are wrong because there seems to be a Tab and two spaces instead of four spaces.
This is breaking the doc build.

Reply:
fixed